src/ducks/billsMatching/Linker/billsToOperation/findNeighboringOperations.spec.js

```diff
@@ -0,0 +1,72 @@
+jest.mock('cozy-konnector-libs')
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.